packages/squiggle-lang/src/rescript/Distributions/SampleSetDist/SampleSetDist_ToPointSet.res

```diff
@@ -70,64 +55,53 @@ let toPointSetDist = (
     ~minDiscreteWeight=minDiscreteToKeep,
   )
 
-  let length = samples->E.A.length->float_of_int
+  let contLength = continuousPart->E.A.length
+  let length = samples->E.A.length
+  let outLength = contLength <= 5 ? length - contLength : length
+  let pointWeight = 1. /. outLength->float_of_int
+
+  let pdf = if contLength <= 5 {
+    // Drop these points; missing mass accounted for by outLength above
+    None
```

Comment:
later on(future story) it would be good to return error

Reply:
No? This happens when almost all points are identified as discrete, and a discrete distribution plus a very small continuous one is perfectly reasonable to have.

Reply:
This is a lot of different ``expect`` calls for one ``test`` call. I imagine we could either split up the ``test`` call, or put the ``test`` call under the ``property`` call.